docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -41,7 +41,7 @@ To create an automated process with user tasks, take the following steps:
 9. Append a gateway to your user task by dragging it onto the dashboard from the palette on the left side of the screen, or by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 10. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
 11. Attach an end event to the two user tasks.
-![gateway example](./img/gateway-example-1.png)
+![gateway example](./img/gateway-example-dinner.png)
```

Comment:
Here you could use bpmn.js instead and add:

`<div bpmn="getting-started-guides/prepare-dinner.bpmn" />`

You could also add callouts to tasks by adding: callouts="DecideDinner,PrepareChicken" 

And then explain this by refering to the callout here in the text:

`<span className="callout">1</span>`

But not urgent at the moment I think - just wanted to flag for awareness.

Reply:
I'm not familiar with implementing BPMN attachments in Markdown. Have I inserted this correctly in the latest commit? On build it doesn't look quite right.

Reply:
@berndruecker I can also just cherry pick out this commit (where I added the BPMN) and you can implement the BPMN in a subsequent PR. Just let me know what's best.